src/main/java/com/study/realworld/secure/GUIDGenerator.java

```diff
@@ -0,0 +1,33 @@
+package com.study.realworld.secure;
+
+import java.util.Random;
+
+public class GUIDGenerator implements IGenerator {
+	
+	protected final int _length;
+	protected final Random _random;
```

Comment:
```suggestion
	protected final int length;
	protected final Random random;
```
변수명에 언더바를 쓰지 않는 것이 컨벤션입니다 :)

Reply:
바뀐 상수들에 대해서 런 이슈있었던 부분들 새로 수정하여 커밋했습니다!